steamweb.py

```diff
@@ -150,22 +157,36 @@ def login(self, captchagid='-1', captcha_text='', emailauth='', emailsteamid='',
                 self._save_cookies()
                 return True
 
-            elif data.get('captcha_needed', False) and data.get('captcha_gid', '-1') != '-1':
+            elif data.get('captcha_needed') and data.get('captcha_gid') != '-1':
```

Comment:
Correct me if I'm wrong but this will return None for `data.get('captcha_gid')` if data has no key 'captcha_gid' so the test will fail. I would also leave the first one to default to False because the value should be True/False not True/False/None.